corehq/motech/dhis2/tasks.py

```diff
@@ -80,6 +81,14 @@ def send_dataset(
             dataset = get_dataset(dataset_map, send_date)
             response = requests.post('/api/dataValueSets', json=dataset,
                                      raise_for_status=True)
+        except DatabaseError:
+            return {
+                'success': False,
+                'error': _('There was an error retrieving some UCR data. '
+                           'Try contacting support to help resolve this issue.'),
+                'traceback': traceback.format_exc(),
```

Comment:
I don't think we want to send tracebacks to the user. You can use `notify_exception` to send the error to sentry.

Reply:
I removed the traceback key. Turns out it was redundant.